ssr/src/page/post_view/mod.rs

```diff
@@ -340,3 +342,16 @@ pub fn PostView() -> impl IntoView {
         </Suspense>
     }
 }
+
+#[component]
+pub fn PostView() -> impl IntoView {
```

Comment:
You can instead revert https://github.com/go-bazzinga/hot-or-not-web-leptos-ssr/commit/701983cd269715980dd2197fed74d45eabfbd6ae